shard.yml

```
@@ -4,7 +4,7 @@ version: 0.1.0
 authors:
   - Johannes Müller <straightshoota@gmail.com>
 
-crystal: 0.35.0
+crystal: '>= 0.35.0, < 2.0.0'
```

```suggestion
crystal: '>= 0.35.0'
```
I don't think adding an upper bound restriction is a good idea. For all I know, this shard should be compatible with Crystal 2.0 (whenever that gets released). Until there is a known incompatibility, I don't like to impose any artificial restrictions.

The resulting change from `crystal: 0.35.0` to `crystal: >= 0.35.0` is semantically equivalent. The latter format is preferred, though. So I think it's still a good change.